src/commands/validators/index.ts

```diff
@@ -0,0 +1,88 @@
+import { Command } from "commander";
+import { ValidatorsAction } from "./validators";
+
+export function initializeValidatorCommands(program: Command) {
+  const validatorsAction = new ValidatorsAction();
+
+  const validatorsCommand = program
+    .command("validators")
+    .description("Manage validator operations");
+
+  validatorsCommand
+    .command("get")
+
+    .description("Retrieve details of a specific validator or all validators")
+    .option("--address <validatorAddress>", "The address of the validator to retrieve (omit to retrieve all validators)")
+    .action(async (options) => {
+      await validatorsAction.getValidator({ address: options.address });
+    });
+
+  validatorsCommand
+    .command("delete")
+    .description("Delete a specific validator or all validators")
+    .option("--address <validatorAddress>", "The address of the validator to delete (omit to delete all validators)")
+    .action(async (options) => {
+      await validatorsAction.deleteValidator({ address: options.address });
+    });
+
+  validatorsCommand
+    .command("count")
+    .description("Count all validators")
+    .action(async () => {
+      await validatorsAction.countValidators();
+    });
+
+  validatorsCommand
+    .command("update <validatorAddress>")
+    .description("Update a validator's details")
+    .option("--stake <stake>", "New stake for the validator")
+    .option("--provider <provider>", "New provider for the validator")
+    .option("--model <model>", "New model for the validator")
+    .option("--config <config>", "New JSON config for the validator")
+    .action(async (validatorAddress, options) => {
+      await validatorsAction.updateValidator({
+        address: validatorAddress,
+        stake: options.stake,
+        provider: options.provider,
+        model: options.model,
+        config: options.config,
+      });
+    });
+
+  validatorsCommand
+    .command("create-random")
+    .description("Create random validators")
+    .option("--count <count>", "Number of validators to create", "1") // Default to "1"
+    .option(
```

Comment:
@epsjunior we are also missing the --models option to match the endpoint full parameters: https://github.com/yeagerai/genlayer-studio/blob/main/backend/protocol_rpc/endpoints.py#L190